### DISCUSSION TITLE
Тазер

### DISCUSSION THREAD

**Discussion by Asdasdasdasdsaddczx2** [octothorp-team/public-dobrograd#852] [Предложения]
Ура!! Снова я с глупыми идеями! И так не будем тянуть кота за хвост. Моя идея несёт в себе:

У алкоголя есть эффект головокружения, что же я предлогаю сделать? А именно увиличить фриз игрока в регдолле, ибо тазер реально мощный и если у человека имеются противопоказания к сердцу, он может даже умереть, так вот а так же после эффекта думаю стоит сделать этот эффект головокружения на 5-6 секунд, тоесть что дает понять человеку, что тазер реально серьезная штука. 

Спасибо за внимание! :D:D:D:D:D:D:D

**Comment by sergchuk** [comment]
Согласен, еще можно с бин бэком сделать сильную тряску после выстрела, потомучто бин бэк очень мощная штука, он может убить если стрелять в голову
